tstest/natlab/natlab.go

```diff
@@ -31,6 +31,18 @@ import (
 
 var traceOn, _ = strconv.ParseBool(os.Getenv("NATLAB_TRACE"))
 
+// TODO(danderson): this makes it impossible to parallelize natlab
```

Comment:
This also probably violates the sync.WaitGroup rules about all Adds having to happen before a Wait. If an Add and Wait happen concurrently, things explode: https://golang.org/pkg/sync/#WaitGroup.Add